### PR TITLE
benchmark: New package for the Google Benchmark library

### PR DESCRIPTION
**Pull request by eschnett**

### CODE REVIEW

File: var/spack/repos/builtin/packages/benchmark/package.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+class Benchmark(CMakePackage):
+    """A microbenchmark support library"""
+
+    homepage = "https://github.com/google/benchmark"
+    url      = "https://github.com/google/benchmark/archive/v1.1.0.tar.gz"
+
+    version('1.1.0', '66b2a23076cf70739525be0092fc3ae3')
+    version('1.0.0', '1474ff826f8cd68067258db75a0835b8')
+
+    def cmake_args(self):
```
Comment (davydden) [inline var/spack/repos/builtin/packages/benchmark/package.py]: just remove this if it does not do anything.

Comment (eschnett) [inline var/spack/repos/builtin/packages/benchmark/package.py]: Done.

**Comment by alalazo** [comment]
Didn't we have already [gbenchmark](https://github.com/LLNL/spack/blob/develop/var/spack/repos/builtin/packages/gbenchmark/package.py)? Is this a duplicate package?

**Comment by eschnett** [comment]
Yes, this is a duplicate package. I didn't expect the package to be named `gbenchmark`since that name doesn't follow from the package name. My `benchmark` package should be removed again.

**Pull request merged**